src/gerbil/prelude/core.ss

```diff
@@ -159,16 +159,20 @@ package: gerbil
     string-split string-join string-empty? string-prefix?
     string->keyword keyword->string make-uninterned-keyword
     symbol->keyword keyword->symbol
+    ;; c3 linearization
+    c3-linearize not-null? remove-nulls append-reverse
     ;; MOP
     type-descriptor?
-    struct-type?
-    class-type?
-    make-struct-type
+    type-id type-descriptor-precedence-all-slots type-descriptor-precedence-list type-descriptor-alist type-descriptor-constructor type-descriptor-slot-table type-descriptor-methods
+    type-final? type-struct? struct-type? class-type?
+    make-struct-type make-struct-type*
     make-struct-predicate
-    make-struct-field-accessor
-    make-struct-field-mutator
-    make-struct-field-unchecked-accessor
-    make-struct-field-unchecked-mutator
+    make-struct-field-accessor make-struct-field-accessor*
+    make-struct-field-mutator make-struct-field-mutator*
+    make-struct-field-unchecked-accessor make-struct-field-unchecked-accessor*
+    make-struct-field-unchecked-mutator make-struct-field-unchecked-mutator*
+    base-struct/1 base-struct/2 base-struct/list base-struct* base-struct
```

Comment:
why are these being added to the prelude? are they useful in general? If they are details they can be imported through `:gerbil/runtime/mop` as needed.